governance/polls/Vault Compensation Denomination - April 13, 2020.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Vault Compensation Denomination Poll - April 13, 2020
+summary: Signal whether to provide compensation to vault holders in the vault's token (ETH/BAT) or in DAI
+discussion_link: https://forum.makerdao.com/t/1822
```

Comment:
Can we add the discussion link below:
https://forum.makerdao.com/t/compensating-vault-holders-that-liquidated-at-0-bid/1541/85

Reply:
This link is already in `More background discussion`. Or do you want to change the main discussion link in the poll header?